src/detours.cpp

```diff
@@ -1683,7 +1760,8 @@ LONG WINAPI DetourTransactionCommitEx(_Out_opt_ PVOID **pppFailedPointer)
 #endif // DETOURS_ARM
 
 #ifdef DETOURS_ARM64
-            PBYTE pbCode = detour_gen_jmp_immediate(o->pbTarget, NULL, o->pTrampoline->pbDetour);
+            o->pTrampoline->rbIndTarget = (ULONG64)(o->pTrampoline->pbDetour);
```

Comment:
nit: no parens needed on `o->pTrampoline->pbDetour`.

Reply:
Same on next line. I get it, though, `->` is an operator, with such high precedence people don't even see it as such.